### ISSUE TITLE
FastBoot: Cannot read property 'default' of undefined at Module.callback

### ISSUE THREAD

**Issue by josemarluedke** [text-mask/ember-text-mask-addons#12]
Hi there!

When importing `import { createNumberMask } from 'ember-text-mask-addons';` in FastBoot, we get the following error:

```
TypeError: Cannot read property 'default' of undefined at Module.callback (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:92825:123) at Module.exports (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:140:32) at Module._reify (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:173:59) at Module.reify (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:159:27) at Module.exports (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:138:10) at requireModule (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:32:18) at Class._extractDefaultExport (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:92529:20) at Class.resolveOther (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:92226:32) at Class.superWrapper (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:50290:22) at Class.resolveRoute (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:15741:19) at Class.resolve (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:15585:43) at resolve (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:12995:36) at Registry.resolve (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:12454:21) at Registry.resolve (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:12458:55) at factoryFor (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:12005:28) at instantiate (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:12070:19) at lookup (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:11938:17) at Container.lookup (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:11857:14) at Class.lookup (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:42430:33) at C.getHandler (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:37848:34) at C.fetchHandler (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:66868:24) at C.get (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:67043:17) at C.applyToState (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:67441:35) at Router.getTransitionByIntent (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:67497:25) at Router.transitionByIntent (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:67613:36) at doTransition (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:68215:17) at Router.handleURL (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:67657:12) at Class._doURLTransition (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:37614:51) at Class.handleURL (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:37610:19) at Class.visit (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:13554:21) at /Users/josemarluedke/Projects/Temp/text-mask-fastboot/node_modules/fastboot/dist/cjs/ember-app.js:238:28 at tryCatch (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:68840:14) at invokeCallback (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:68855:15) at publish (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:68823:9) at /Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:48280:16 at invokeWithOnError (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:10898:16) at Queue.flush (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:10957:9) at DeferredActionQueues.flush (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:11081:15) at Backburner.end (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:11151:23) at Timeout._onTimeout (/Users/josemarluedke/Projects/Temp/text-mask-fastboot/dist/fastboot/vendor.js:11717:16) at ontimeout (timers.js:365:14) at tryOnTimeout (timers.js:237:5) at Timer.listOnTimeout (timers.js:207:5)
```

This only happens in FastBoot mode, when executing the same code in the browser, it works fine.

I have created a reproduction repo with this error, it lives at https://github.com/josemarluedke/text-mask-fastboot. 

To test it local, just clone the repo, install the deps and execute: `ember fastboot`.

If you provide any guidance on how to fix it, it would be very helpful. Thank you.

**Comment by lozjackson** [comment]
Hi, I will take a look and get back to you

**Comment by lozjackson** [comment]
Fixed in v2.1.0
